src/assets/css/librarybrowser.css

```diff
@@ -529,11 +529,14 @@
     align-items: center;
 }
 
+.parentName > a:nth-child(3) {
```

Comment:
this feels like a dirty hack...

Reply:
Any other quick & non-hacky solution without rewriting a lot of the JS? As I said before, I would like this quick for 10.5 (so later on we can focus on it deeper) but if everyone doesn't agree, I don't have any problem in doing that fine for 10.6.

Reply:
See my other review comment. Separate both (only on mobile) and add some class or id to the new element if needed.

Here, you're changing the rule for all layouts, where the goal was to change it only for mobile :)

As a general rule of thumb, :nth-child is to be avoided since any new element or layout change will completely break the rule :/

Reply:
The problem is that if we add the <br> tag, in Desktop would overflow a lot, that's why I applied it to all layouts. If I have an 100% reliable way of checking the target device in JS, I can easily remove this.